decoder/main.go

```diff
@@ -411,10 +416,11 @@ func ClearRemovedForts(ctx context.Context, dbDetails db.DbDetails,
 						continue
 					}
 					log.Infof("Cleared old Stop(s) in cell %d: %v", cellId, toClear)
-					//TODO send webhook
+					CreateFortWebhooks(ctx, dbDetails, toClear, POKESTOP, REMOVAL)
```

Comment:
@jfberry I don't think it's accessible from here (getPokestopRecord)... I can try again tomorrow

Reply:
it's in decoder as well

Reply:
Makes sense, maybe eyes were to tired haha